File: backend/dalathasfarm/src/main/java/com/example/dalathasfarm/dtos/FeedbackDto.java
package com.example.dalathasfarm.dtos;

import lombok.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
public class FeedbackDto {
    private int userId;
    private String content;
    private int star;
    private int productId;
}

File: android/lib/dtos/feed_back_dto.dart
class FeedbackDto {
  final int userId;
  final String content;
  final int star;
  final int productId;

  FeedbackDto({
    required this.userId,
    required this.content,
    required this.star,
    required this.productId,
  });

  Map<String, dynamic> toJson() {
    return {
        'userId': userId,
        'content': content,
        'star': star,
        'productId': productId, 
    };
  }
}
